docs/components/dartpad/top-app-bar/contextual/main.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      debugShowCheckedModeBanner: false,
```

Comment:
I'd remove debugShowCheckedModeBanner since it isn't necessary for the demo

Reply:
Done